### DISCUSSION TITLE
Is it possible to expose `isNewUser` via `signInWithApple`?

### DISCUSSION THREAD

**Discussion by howlettga** [capawesome-team/capacitor-firebase#698] [Ideas]
I see an existing conversation discussed this a couple years ago: https://github.com/capawesome-team/capacitor-firebase/discussions/261

Is it a possibility to expose the `isNewUser` response from Firebase when using `signInWithApple` with `skipNativeAuth=true` or is this a limitation outside of this plugins control? It seems like `displayName` is exposed in the sign in request even with `skipNativeAuth`, so maybe `isNewUser` could be too? I realize this would be coming from Firebase, not Apple. It is a useful boolean to know whether to route to an onboarding view or not.

**Comment by robingenz** [comment]
No, this is not possible as this information is only available after authentication.
